<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 5.0 (Python 3.7.0 on win32)" generated="20220430 23:49:24.668" rpa="false" schemaversion="3">
<suite id="s1" name="demoProjects" source="D:\demoProjects">
<suite id="s1-s1" name="Testcases" source="D:\demoProjects\testcases">
<suite id="s1-s1-s1" name="GUI scripts" source="D:\demoProjects\testcases\GUI_scripts.robot">
<test id="s1-s1-s1-t1" name="test" line="136">
<kw name="Comment" library="BuiltIn">
<arg>log</arg>
<arg>@{nums}[0]</arg>
<doc>Displays the given messages in the log file as keyword arguments.</doc>
<status status="PASS" starttime="20220430 23:49:25.299" endtime="20220430 23:49:25.299"/>
</kw>
<kw name="Comment" library="BuiltIn">
<arg>log</arg>
<arg>@{nums}</arg>
<doc>Displays the given messages in the log file as keyword arguments.</doc>
<status status="PASS" starttime="20220430 23:49:25.299" endtime="20220430 23:49:25.299"/>
</kw>
<kw name="Log" library="BuiltIn">
<arg>@{hello}</arg>
<doc>Logs the given message with the given level.</doc>
<msg timestamp="20220430 23:49:25.299" level="FAIL">Invalid log level '6'.</msg>
<status status="FAIL" starttime="20220430 23:49:25.299" endtime="20220430 23:49:25.299"/>
</kw>
<status status="FAIL" starttime="20220430 23:49:25.299" endtime="20220430 23:49:25.299">Invalid log level '6'.</status>
</test>
<status status="FAIL" starttime="20220430 23:49:25.234" endtime="20220430 23:49:25.299"/>
</suite>
<status status="FAIL" starttime="20220430 23:49:24.763" endtime="20220430 23:49:25.312"/>
</suite>
<status status="FAIL" starttime="20220430 23:49:24.732" endtime="20220430 23:49:25.312"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="demoProjects">demoProjects</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1" name="Testcases">demoProjects.Testcases</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1-s1" name="GUI scripts">demoProjects.Testcases.GUI scripts</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20220430 23:49:25.281" level="ERROR">[ DataDriver ] Error in robot file:
  File "D:\demoProjects\testcases\GUI_scripts.robot", line 0</msg>
<msg timestamp="20220430 23:49:25.281" level="ERROR">In source file:
  File "D:/demoProjects/testData/test_user_data.xlsx", line 0</msg>
<msg timestamp="20220430 23:49:25.296" level="ERROR">Calling method '_start_suite' of listener 'DataDriver' failed: FileNotFoundError: File attribute was not a full path. Tried to find D:/demoProjects/testData/test_user_data.xlsx but file does not exist.</msg>
</errors>
</robot>
